internal/controller/streams.go

```diff
@@ -236,6 +272,26 @@ func connectAndStreamCiliumNetworkFlows(logger *zap.SugaredLogger, sm *streamMan
 	return nil
 }
 
+// connectAndStreamFalcoNetworkFlows creates networkFlowsStream client and begins the streaming of network flows.
+func connectAndStreamFalcoNetworkFlows(logger *zap.SugaredLogger, sm *streamManager) error {
+	falcoCtx, falcoCancel := context.WithCancel(context.Background())
+	defer falcoCancel()
+	sendFalcoNetworkFlows, err := sm.streamClient.client.SendKubernetesNetworkFlows(falcoCtx)
```

Comment:
Yeah it is possible but I think we are going to have an advisory to not let that happen. If we have a customer who will have both on their cluster it still should be fine because in order to have falco talk to our operator they'll need to specifically configure it with out service endpoint. (which cannot accidentally happen)